sdk/AWSSDK.CodeAnalysis.sln

```diff
@@ -1,785 +1,785 @@
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.WorkMailMessageFlow.CodeAnalysis", "code-analysis\ServiceAnalysis\WorkMailMessageFlow\AWSSDK.WorkMailMessageFlow.CodeAnalysis.csproj", "{96AA8820-0649-4AE3-92C2-C01F7B30E874}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.WorkSpaces.CodeAnalysis", "code-analysis/ServiceAnalysis/WorkSpaces/AWSSDK.WorkSpaces.CodeAnalysis.csproj", "{59D2738F-B594-4863-B5A7-1ACF80A71885}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.WorkSpaces.CodeAnalysis", "code-analysis\ServiceAnalysis\WorkSpaces\AWSSDK.WorkSpaces.CodeAnalysis.csproj", "{59D2738F-B594-4863-B5A7-1ACF80A71885}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.WorkSpacesThinClient.CodeAnalysis", "code-analysis/ServiceAnalysis/WorkSpacesThinClient/AWSSDK.WorkSpacesThinClient.CodeAnalysis.csproj", "{667D139D-B7A8-4828-BC21-03083A564F18}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.WorkSpacesThinClient.CodeAnalysis", "code-analysis\ServiceAnalysis\WorkSpacesThinClient\AWSSDK.WorkSpacesThinClient.CodeAnalysis.csproj", "{667D139D-B7A8-4828-BC21-03083A564F18}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.WorkSpacesWeb.CodeAnalysis", "code-analysis/ServiceAnalysis/WorkSpacesWeb/AWSSDK.WorkSpacesWeb.CodeAnalysis.csproj", "{4D0C0F30-FEE9-4C5F-AD1B-CABE4E390588}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.WorkSpacesWeb.CodeAnalysis", "code-analysis\ServiceAnalysis\WorkSpacesWeb\AWSSDK.WorkSpacesWeb.CodeAnalysis.csproj", "{4D0C0F30-FEE9-4C5F-AD1B-CABE4E390588}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.XRay.CodeAnalysis", "code-analysis/ServiceAnalysis/XRay/AWSSDK.XRay.CodeAnalysis.csproj", "{5D84DBCB-DBE7-41C6-8DC2-D4BC1ABE0BE7}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.XRay.CodeAnalysis", "code-analysis\ServiceAnalysis\XRay\AWSSDK.XRay.CodeAnalysis.csproj", "{5D84DBCB-DBE7-41C6-8DC2-D4BC1ABE0BE7}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SharedAnalysisCode", "code-analysis/SharedAnalysisCode/SharedAnalysisCode.csproj", "{16E2A13F-F4F6-4774-A2D8-AD8127E7EE4F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SharedAnalysisCode", "code-analysis\SharedAnalysisCode\SharedAnalysisCode.csproj", "{16E2A13F-F4F6-4774-A2D8-AD8127E7EE4F}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MockAnalyzer", "code-analysis/MockAnalyzer/MockAnalyzer.csproj", "{9BC3B08B-9F09-444E-AD15-67045FA3AF19}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MockAnalyzer", "code-analysis\MockAnalyzer\MockAnalyzer.csproj", "{9BC3B08B-9F09-444E-AD15-67045FA3AF19}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AnalyzerUnitTests", "code-analysis\AnalyzerUnitTests\AnalyzerUnitTests.csproj", "{FB4B26D0-2E81-4673-9001-D0C859D8CEA1}"
```

Comment:
This file is generated, right? I downloaded the output from the dry-run and this line (`"AnalyzerUnitTests", "code-analysis\AnalyzerUnitTests\AnalyzerUnitTests.csproj"`) was removed.

Reply:
Good catch. Updated the generator.